README.rst

```diff
@@ -17,7 +17,7 @@ Use pip to install!
 
     pip install keen
 
-This client is known to work on Python 2.6, 2.7, 3.2, 3.3, 3.4 and 3.5.
+This client is known to work on Python 2.6, 2.7, 3.2, 3.3, 3.4, 3.5 and 3.6.
```

Comment:
Still listing 3.2 here but it was removed from Travis validation. Is 3.2 support being dropped or is there a reason not to include it in Travis runs?